install/installer/pkg/components/registry-facade/daemonset.go

```diff
@@ -309,6 +309,19 @@ func daemonset(ctx *common.RenderContext) ([]runtime.Object, error) {
 								common.NodeNameEnv(ctx),
 								common.ProxyEnv(&ctx.Config),
 							)),
+							LivenessProbe: &corev1.Probe{
+								ProbeHandler: corev1.ProbeHandler{
+									HTTPGet: &corev1.HTTPGetAction{
+										Path: "/ready",
+										Port: intstr.IntOrString{IntVal: ReadinessPort},
+									},
+								},
+								InitialDelaySeconds: 5,
+								PeriodSeconds:       2,
+								TimeoutSeconds:      2,
+								SuccessThreshold:    1,
+								FailureThreshold:    3,
+							},
```

Comment:
This makes sense to me, nice work, @utam0k !  In other words, this liveness probe will only prevent scheduling of future workspaces (until the label is added back), it won't interrupt workspaces that are actively pulling images on start.

Reply:
@utam0k why are `node-labeler` PeriodSeconds time 2, and `registry-facade` 10? 

The side effect is we'll be more likely (when network conditions are poor) to remove the node label, but not necessarily restart the `registry-facade` container (until more time has passed).

I assume intentional? 

I am going to include @sagor999 , so that he can check if this will have an unintended consequence, and whether that is acceptable or not. 

For example, if when we remove the label, that may cause the autoscaler to provision a new node, because more Re: workspace pods are unscheduable (ultimately, probably a good thing).